Maple2.Server.Game/PacketHandlers/InteractObjectHandler.cs

```diff
@@ -39,7 +40,9 @@ private enum Command : byte {
     private void HandleEnd(GameSession session, IByteReader packet) {
         string entityId = packet.ReadString();
 
-        if (session.Field?.TryGetInteract(entityId, out FieldInteract? interact) == true && interact.React()) {
+        if (entityId.StartsWith("BillBoard") && session.Field?.TryGetAdBalloon(entityId, out FieldInteract? interact) == true && interact.React()) {
```

Comment:
`entityId.StartsWith("BillBoard")` feels very flaky. I don't want to depend on this condition.